src/main/resources/mybatis/navigate.xml

```diff
@@ -137,19 +140,20 @@
 
     <sql id="UT">
         with
-            recursive navigation_results(comid, hydroseq, startflag, stoplength)
-                as (select comid, hydroseq, startflag,
+            recursive navigation_results(comid, hydroseq, startflag, fcode, stoplength)
+                as (select comid, hydroseq, startflag, fcode,
                            pathlength + 0 <if test="null != distance">+ #{distance}</if> stoplength
                       from nhdplus.plusflowlinevaa_np21
                      where comid = #{comid,jdbcType=NUMERIC}
                     union
-                    select x.comid, x.hydroseq, x.startflag, navigation_results.stoplength
+                    select x.comid, x.hydroseq, x.startflag, x.fcode, navigation_results.stoplength
                       from nhdplus.plusflowlinevaa_np21 x,
                            navigation_results
                      where navigation_results.startflag != 1 and
                            (x.dnhydroseq = navigation_results.hydroseq or
                             (x.dnminorhyd != 0 and
-                             x.dnminorhyd = navigation_results.hydroseq))
+                             x.dnminorhyd = navigation_results.hydroseq)) and
+                            x.fcode != 56600
```

Comment:
This is kind of a magic number. Can we create a variable called "coastal_fcode = 56600" then use that variable? And/or please comment this line describing what this is doing.

Reply:
Oh, good catch.  That should probably be a constant, or even a configuration setting if we think it may change frequently.

Ethan, I know you ran into some challenges setting up the unit test, but we can figure it out together.

Reply:
It won't change. I'm ok with it being hard coded but it should be given an expressive variable name.